README.md

```diff
@@ -28,14 +28,17 @@ them with your package manager.
 Else you can install them manually
 
 ### Meson
+You need to install python 3 and pip. On debian and ubuntu, they are provided by the package python3-pip
 
 ```
-pip install meson --user # Install Meson
+pip3 install meson --user # Install Meson
+cp ~/.local/bin/meson <a_directory_in_your_path>
```

Comment:
Apparently, it has been added for next version of Debian (Stretch) : https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=820856
For Ubuntu, it has been added in 16.10, and backported on 16.04 : https://bugs.launchpad.net/ubuntu/+source/bash/+bug/1588562
So, If we do not want to officially support Debian 8 or earlier versions of Ubuntu, it's not worth mentioning, I guess.